test/helpers/add-files.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs');
```

Comment:
Please add `'use strict'` statements + a new line in all new files (this is not enforced in lint because the files are treated as modules)